Extractor/FrontendExtractor.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Bazinga\Bundle\JsTranslationBundle\Extractor;
+
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Translation\Extractor\AbstractFileExtractor;
+use Symfony\Component\Translation\MessageCatalogue;
+use Symfony\Component\Translation\Extractor\ExtractorInterface;
+
+final class FrontendExtractor extends AbstractFileExtractor implements ExtractorInterface
+{
+    const FUNCTION_STRING_ARGUMENT_REGEX = '\s?[\'"]([^"\'),]+)[\'"]\s?';
```

Comment:
The final space matching is useless (except to increase backtracking due to being non-possessive), as you don't care about what comes next